index.html

```diff
@@ -1,25 +1,31 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css" />
         <title>Landing Page</title>
     </head>
 
     <body>
         <div class="header">
             <div class="top">
                 <h1>Header Logo</h1>
-                <a href="#"> header link one</a>
-                <a href="#"> header link two</a>
-                <a href="#"> header link three</a>
+                <nav class="nav">
+                    <a href="#"> header link one</a>
+                    <a href="#"> header link two</a>
+                    <a href="#"> header link three</a>
+                </nav>
             </div>
-            <div class="topHalfSplit">
+            <div class="top-half-split">
                 <div class="splitChild">
                     <h2>This website is awesome</h2>
-                    <p class="secondaryText">This website has some subtext that goes here under the main title. It's a smaller font and the color is lower contrast</p>
-                    <button class="signupbutton">Sign up</button>
+                    <p class="secondaryText">
+                        This website has some subtext that goes here under the
+                        main title. It's a smaller font and the color is lower
+                        contrast
+                    </p>
+                    <button type="button" class="button">Sign up</button>
```

Comment:
Added `type="button"` attribute to indicate this is a general purpose button, not a form submission button. 

However, buttons should always be used for some kind of "action" which generally means submitting a form or some kind of Javascript trigger. In this case, I assume the "button" will actually link to a sign-up page in which case it should be an `<a href="" />` styled to look like a button.